tasks/Restaurant.tex

```diff
@@ -0,0 +1,90 @@
+There are \emph{two} robots helping clients in the restaurant at the same time.
+
+% \subsection{Focus}
+% This test focuses on online mapping, safe navigation in previously unknown environments, gesture detection, human-robot interaction, and manipulation in a real environment.
+
+\subsection{Main Goal}
+Taking two orders from the quests and serving the quests with the requested orders. Before taking the order, the robot can be guided by a \textit{Professional Waiter} (member of the TC). During delivering phase, the robot should finds its own way.
+
+\noindent\textbf{Reward:} 1000pts, 350 regular per order + 150 bonus per order
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Detection of quest requesting service by gesture or calling (75pts)
+	\item Arriving at table of the guest without guidance (75pts)
+\end{enumerate}
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Location:} A real restaurant fully equipped with a \enquote{Professional Barman} i.e.~the operator and at least three tables with \enquote{Professional Clients}.
+    \item \textbf{Kitchen-bar:} The \textit{Kitchen-bar} will be a table located at the restaurant's kitchen, next to the place where the robot started. It has the following setup.
+	\begin{itemize}[nosep]
+		\item \textbf{Barman:} A \textit{Professional Barman} (member of the TC) will be at the other side of the \textit{Kitchen-bar} to take the order provided by the robot and serve it in the official tray.
+		\item \textbf{Items:} Items will be located on the Kitchen-bar next to the \textit{Professional Barman}.
+	\end{itemize}
+\end{itemize}
+
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Safety!} This test takes place in a public area. That is, there may be people standing, sitting or walking around the area throughout the test. The robot is expected to not even slightly touch anything and is immediately stopped in case of danger.
+
+	\item \textbf{Referees and guidance:} For safety reasons, the referees in this test are TC members. One of the referees follows the robot and is always in reach of the emergency button.
+
+    \item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per order as follows.
+        \begin{itemize}[nosep]
+          \item Needing guidance during the deliver phase causes a reduction of 40\% in scoring.
+          \item Needing assistance for picking up an item/the tray causes a reduction of 10\% in scoring.
+          \item Have a human place the item in a basket causes a reduction of 20\% in scoring.
+          \item Have the quest take the item out of the basket causes a reduction of 20\% in scoring.
+          \item Asking the barman at which side the \textit{Kitchen-bar} is causes a reduction of 5\% in the total scoring.
+        \end{itemize}
+
+	\item \textbf{Start:} There is no fixed start signal in this test, it starts when both robots are ready (though within a reasonable time).
```

Comment:
No fixed start what so ever? usually we have a queue of Teams and there will be some sort of way to get them to the starting location and have them start

Reply:
I would say that is preparation. No fixed start like opening a door. (This is just copied from the old Rulebook)